kafka-ui-react-app/src/components/Topics/Topic/Messages/Filters/Filters.tsx

```diff
@@ -109,7 +109,11 @@ const Filters: React.FC<FiltersProps> = ({
   );
 
   const [currentSeekType, setCurrentSeekType] = React.useState<SeekType>(
-    (searchParams.get('seekType') as SeekType) || SeekType.OFFSET
+    SeekTypeOptions.map(({ value }) => value).includes(
```

Comment:
i think we can use find here instead of iterating over the array twice , 
[MDN doc for find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)